docs/rfc/2022-07-05-mo-cluster.md

```diff
@@ -87,6 +87,46 @@ It is worth noting that `MatrixOneCluster` object is not necessary to create a m
 - advanced user can ignore `MatrixOneCluster` object perform fine-grained control over `*Set` directly;
 - it is trivial to support heterogenous cluster: one simply deploy several `*Set` objects with different Pod specification;
 
+An overview of the deployment topology:
+
+```mermaid
+flowchart TD
+  subgraph k8s
+    subgraph Cluster
+    direction TB
+    LoadBalancer
+    subgraph CN
+      direction LR
+      CN-0 o--o|MPP| CN-1
+      CN-1 --> |scale-out| CN-2
+      style CN-2 fill:#bbf,stroke:#f66,stroke-width:2px,color:#fff,stroke-dasharray: 5 5
+    end
+    subgraph DN
+      direction RL
+      DN-1 o--o |RPC| DN-0
+						DN-2 --> |standby| DN-0
+						DN-2 --> |standby| DN-1
```

Comment:
In my understanding, it's each R/W-DN can have one or more standby-DNs, rather than a standby-DN belonging to more than one R/W-DN.

Reply:
I think so, the graph is a demonstration at pod level: actually for current version we don't support RO DN replica as a standby, an empty Pod could act as resource level standby that can take over the replica from DN-0 or DN-1, based on the decision of HA keeper.

Reply:
That's fine